src/Apps/Order/Components/Summary.tsx

```diff
@@ -0,0 +1,37 @@
+import Colors from "Assets/Colors"
+import { unica } from "Assets/Fonts"
```

Comment:
So instead of these you'll want to use the components from Palette.... which I need to write some good documentation for, ha. 

```typescript
import { Sans } from "@artsy/palette"
```
So the `Sans` component maps to the values in our [typography design specs](https://www.notion.so/artsy/Typography-d1f9f6731f3d47c78003d6d016c30221). 

Instead of building the `Text` component use...

```jsx
<Sans size="2" color="black60">
...
</Sans>
```

If you're using Zeplin, I wrote an [extension](https://github.com/artsy/palette-zeplin-extension) to help pick out the right typography component.